in_toto/verifylib.py

```diff
@@ -593,4 +593,84 @@ def verify_all_item_rules(items, links, target_links=None):
         link.products, target_links)
 
 
+def in_toto_verify(layout_path, layout_key_paths):
+  """
+  <Purpose>
+    Does entire in-toto supply chain verification of a final product
+    by performing the following actions:
+        1.  Load root layout as specified by 'layout_path'
+        2.  Load root layout signing key(s) (project owner public key(s)) as
+            specified by 'layout_key_paths'
+        3.  Verify layout signature(s)
+        4.  Verify layout expiration
+        5.  Load link metadata for every Step defined in the layout
+            NOTE: in_toto_verify searches for link metadata files by their step
+            name plus the extension '.link' in the current directory.
+            E.g. a step of name 'foo' is expected to be found in the current
+            directory at 'foo.link'.
```

Comment:
Not yet. The keyid postfix will be in PR #63 or #64. :)  Which will also have to update this function (plus docstrings).